extra_worker.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+data:
+  password: {{ ipmi_password | base64 }}
+  username: {{ ipmi_user | base64 }}
+kind: Secret
+metadata:
+  manager: baremetal-operator
+  name: {{ cluster }}-worker-1-bmc-secret
+  namespace: openshift-machine-api
+type: Opaque
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: {{ cluster }}-worker-1
```

Comment:
is this an example? cause what happen if more than one? need to be worker-2, etc.

Reply:
there is a flag, virtual_workers_deploy set to False by default, so that you can actually deploy manually your worker after initial install time.
this is there only for lab purposes, since i believe it could be good to do this manual worker thing as an extra lab step